pkg/.import-restrictions

```diff
@@ -7,7 +7,8 @@
 		],
 		"ForbiddenPrefixes": [
 			"k8s.io/kubernetes/cmd",
-			"github.com/ghodss/yaml"
+			"github.com/ghodss/yaml",
+			"github.com/ishidawataru/sctp"			
```

Comment:
So, we need this module to use in agnhost to do the SCTP testing, but per the KEP, kubernetes itself should never be creating SCTP sockets. So I added it to the import-restrictions here to keep anyone else from using it...